### ISSUE TITLE
Decrypt fails on openssl 1.1 without -md md5

### ISSUE THREAD

**Issue by samengstrom** [pavjacko/renative#377]
**Describe the bug**
Decrypt fails on openssl 1.1 without -md md5

**To Reproduce**
Try to run rnv with decrypt on a system with openssl 1.1 (e.g. Ubuntu 18). You'll see:
```
*** WARNING : deprecated key derivation used.
Using -iter or -pbkdf2 would be better.
bad decrypt
139781279515072:error:06065064:digital envelope routines:EVP_DecryptFinal_ex:bad decrypt:../crypto/evp/evp_enc.c:537:`
```

**Expected behavior**
Decrypt should work

**Desktop (please complete the following information):**
 - OS Ubuntu 18.04
 - Node Version v10.13.0
 - RNV Version 0.28.12-alpha.10

**Additional context**
The default digest has changed in newer openssls to sha256 so you have to specify md5 with -md md5. The switch might work even in older versions but I haven't tested it.

**Comment by pavjacko** [comment]
@samengstrom  this should be now fixed in `0.28.17-alpha.10`

**Comment by samengstrom** [comment]
Great, thanks!
